src/Pact/Eval.hs

```diff
@@ -270,13 +271,19 @@ eval (TModule (MDInterface m) bod i) =
     oldI <- readRow i Modules $ _interfaceName mangledI
     case oldI of
       Nothing -> return ()
-      Just old -> evalError i $ "Existing interface found: " <> pretty old
+      Just _old -> evalError i $ "Existing interface found (interfaces cannot be upgraded)"
```

Comment:
do we want to just `Just {}` so we don't have to bind a variable here? (Also, side note, we should _definitely_ have this on our radar for documentation)